AdminOpsGuide/dbrepl.rst

```diff
@@ -963,7 +965,7 @@ Here is the code:
    $ydb_dist/mumps -r ^GDE @gdemsr
    $ydb_dist/mupip create
 
-gdemsr contains:
+The file gdemsr contains:
```

Comment:
I found some answers to my previous comment. This change (adding "The file") is what caused "gdemsr" section to disappear. Needs to be undone. You can see the difference in the rendered document.